dask/dataframe/tests/test_dataframe.py

```diff
@@ -2415,9 +2423,11 @@ def test_groupby_multilevel_info():
 
     buf = StringIO()
     g.info(buf, verbose=False)
-    assert buf.getvalue() == unicode("""<class 'dask.dataframe.core.DataFrame'>
-Columns: 2 entries, (C, count) to (C, sum)
-dtypes: int64(2)""")
+    expected = unicode(textwrap.dedent("""\
+    <class 'dask.dataframe.core.DataFrame'>
+    Columns: 2 entries, ('C', 'count') to ('C', 'sum')
```

Comment:
This is slightly different. The values are `'C'` instead of `C`, but I don't think it's worth re-implementing pandas' version 100%.